golang/ecommerce/order/controller/services.go

```diff
@@ -0,0 +1,72 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func GetOrders(w http.ResponseWriter, r *http.Request) {
+	db := Connect()
+
+	var order []Order
+	id := mux.Vars(r)["user_id"]
+
+	err := db.Where("user_id = ?", id).Find(&order).Error
+	if err != nil {
+		panic(err)
+	}
+	json.NewEncoder(w).Encode(order)
+	db.Close()
+}
+
+func CreateOrder(w http.ResponseWriter, r *http.Request) {
+	db := Connect()
```

Comment:
The database connection should only be opened once in the app lifetime - in the `main` function before the server is started. The connection pool can be reused for all API handlers and should be injected as a dependency.